### PR TITLE
feat(service-action-toolbar): update deploy redeploy color

### PR DESCRIPTION
**Pull request by RemiBonnet**
# What does this PR do?

https://qovery.atlassian.net/jira/software/projects/FRT/boards/23?selectedIssue=FRT-1083

Screenshot 2024-04-19 at 11 15 02

---

## PR Checklist

- [ ] This PR introduces breaking change(s) and has been labeled as such
- [ ] This PR introduces new store changes
- [ ] I have followed the library pattern i.e `feature`, `ui`, `data`, `utils`
- [x] I made sure the code is type safe (no any)

### CODE REVIEW

**Comment by RemiBonnet** [comment]
[Qovery Preview](https://www.qovery.com)
---

A preview environment was automatically created via Qovery.
Click on the link below to follow its deployment and use it.
:point_right: [\[PR\] my-env - feat(service-action-toolbar): update deploy redeploy color - 2024-04-19T09:15:15Z](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/e773b890-e6b2-4406-93a9-71f9a7ab6d73/environment/da9c5097-1f41-445a-bd7d-5a8569a7c1f1/services/general)

 Another comment will be posted when deployments are finished

**Comment by RemiBonnet** [comment]
[Qovery Preview](https://www.qovery.com)
---

Your preview environment failed to be deployed !
Click on the link below to check the deployment logs.
:point_right: [\[PR\] my-env - feat(service-action-toolbar): update deploy redeploy color - 2024-04-19T09:15:15Z](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/e773b890-e6b2-4406-93a9-71f9a7ab6d73/environment/da9c5097-1f41-445a-bd7d-5a8569a7c1f1/logs)

File: libs/shared/ui/src/lib/components/action-toolbar/action-toolbar.tsx [hunk 1]
```diff
@@ -53,6 +54,22 @@ const toolbarButtonVariants = cva([], {
...
+        'rounded-l',
+        'border',
+        'border-yellow-600',
+        'hover:bg-neutral-150',
+        'text-yellow-600',
+        'data-[state=open]:bg-neutral-150',
+        'data-[state=open]:text-yellow-600',
+        'text-sm',
+        'outline-0',
+        'min-w-[36px]',
```
[the remark this answers is not in the document]
Comment (RemiBonnet) [reply libs/shared/ui/src/lib/components/action-toolbar/action-toolbar.tsx:74]: https://github.com/Qovery/console/pull/1294/commits/b965d66e2588aa14cccca7df191b9b3daca602ea

**Review by ctjhoa — APPROVED** [review]

File: libs/shared/ui/src/lib/components/dropdown-menu/dropdown-menu.tsx [hunk 2]
```diff
@@ -38,6 +39,11 @@ const dropdownMenuItemVariants = cva(
         disabled: false,
         className: ['hover:bg-red-50', 'text-neutral-350', 'text-red-600'],
```
[the remark this answers is not in the document]
Comment (RemiBonnet) [reply libs/shared/ui/src/lib/components/dropdown-menu/dropdown-menu.tsx]: https://github.com/Qovery/console/pull/1294/commits/cb447fd33ddedf2ae4bf41060dfeb6dd08059837

**Pull request merged**